### PR TITLE
Add Purdue

### PR DESCRIPTION
**Pull request by Sunit-Kulkarni**
Added Purdue University

### CODE REVIEW

**Comment by brycx** [comment]
Hi @Sunit-Kulkarni, you seem to also have included Carnegie Mellon University from #3305. Could you either close that PR or correct this one?

**Comment by Sunit-Kulkarni** [comment]
Great Catch, just removed it

File: _data/education.yml [hunk 1]
```diff
@@ -253,6 +253,15 @@ websites:
       hardware: Yes
       doc: https://www.identity.psu.edu/services/authentication-services/two-factor/
 
+    - name: Purdue University
+      url: http://www.purdue.edu/
+      img: purdue.png
+      tfa: Yes
+      phone: Yes
```
Comment (mxxcon) [inline _data/education.yml]: I looked through their docs site and I don't see mention of them supporting phone call. Is this accurate?

Comment (Sunit-Kulkarni) [reply _data/education.yml]: I know duo allows the option to, but if their docs don't explicitly say it then it should be removed

**Comment by Carlgo11** [comment]
(Cleaned up the PR and removed phone tag.)

**Review by mxxcon — DISMISSED** [review]

**Review by brycx — APPROVED** [review]

**Comment by Carlgo11** [comment]
psst @mxxcon ☝️

**Review by mxxcon — APPROVED** [review]

**Pull request merged**